client/src/main/java/cloud/xline/client/KvClient.java

```diff
@@ -0,0 +1,119 @@
+     *
+     * @throws InvalidProtocolBufferException on deserialize error
+     * @throws CommandExecutionException on command execution error
+     * @throws StatusRuntimeException on server error
+     */
+    public PutResponse put(PutRequest req)
+            throws InvalidProtocolBufferException, CommandExecutionException {
+        KeyRange key = KeyRange.newBuilder().setKey(req.getKey()).setRangeEnd(req.getKey()).build();
+        RequestWithToken req_token =
+                RequestWithToken.newBuilder().setToken(this.token).setPutRequest(req).build();
+        Command cmd = Command.newBuilder().addKeys(0, key).setRequest(req_token).build();
+        CommandResponse res = this.curpClient.propose(cmd, true);
+        return res.getPutResponse();
+    }
+
+    /**
+     * Delete request
+     *
+     * @throws InvalidProtocolBufferException on deserialize error
+     * @throws CommandExecutionException on command execution error
+     * @throws StatusRuntimeException on server error
+     */
+    public DeleteRangeResponse delete(DeleteRangeRequest req)
+            throws InvalidProtocolBufferException, CommandExecutionException {
+        KeyRange key =
+                KeyRange.newBuilder().setKey(req.getKey()).setRangeEnd(req.getRangeEnd()).build();
+        RequestWithToken req_token =
+                RequestWithToken.newBuilder()
+                        .setToken(this.token)
+                        .setDeleteRangeRequest(req)
+                        .build();
+        Command cmd = Command.newBuilder().addKeys(0, key).setRequest(req_token).build();
+        CommandResponse res = this.curpClient.propose(cmd, true);
+        return res.getDeleteRangeResponse();
+    }
+
+    /**
+     * Txn request
+     *
+     * @throws InvalidProtocolBufferException on deserialize error
+     * @throws CommandExecutionException on command execution error
+     * @throws StatusRuntimeException on server error
+     */
+    public TxnResponse txn(TxnRequest req)
+            throws InvalidProtocolBufferException, CommandExecutionException {
+        List<KeyRange> keyRanges = new ArrayList<>();
+        for (Compare cmp : req.getCompareList()) {
+            KeyRange key =
+                    KeyRange.newBuilder().setKey(cmp.getKey()).setRangeEnd(cmp.getValue()).build();
+            keyRanges.add(key);
+        }
+        RequestWithToken req_token =
+                RequestWithToken.newBuilder().setToken(this.token).setTxnRequest(req).build();
+        Command cmd = Command.newBuilder().addAllKeys(keyRanges).setRequest(req_token).build();
+        CommandResponse res = this.curpClient.propose(cmd, true);
+        return res.getTxnResponse();
+    }
+
+    /**
+     * Compact request
+     *
+     * @throws InvalidProtocolBufferException on deserialize error
+     * @throws CommandExecutionException on command execution error
+     * @throws StatusRuntimeException on server error
+     */
+    public void compact(CompactionRequest req)
```

Comment:
The compact method returns void I think is ok, but our rust clinet returns `CompactionResponse`, is it necessary to be consistent with it?

Reply:
The current client (blocking client) is prepared for clojure (jepsen). The user does not use it.